portfolio/src/main/webapp/origami.js

```diff
@@ -1,25 +1,25 @@
 // number of images in the origami folder, hardcoded :(
-const N = 20;
+const NUMBER_OF_IMAGES = 20;
 const PATH_PREFIX = 'images/origami/';
 const CARD_CN = 'gallery__card';
 
 function loadOrigamiToGallery() {
     const gallery = document.getElementById('gallery');
     // 1-indexed for easier reference
     const origami = [
-        '', 'bird', 'tulip', 'crab', 'flower', 'frog', 'boat', 'swan', 'raccoon', 'fox', 'dog', 'cat',
+        'bird', 'tulip', 'crab', 'flower', 'frog', 'boat', 'swan', 'raccoon', 'fox', 'dog', 'cat',
         'butterfly', 'mouse', 'penguin', 'pinwheel', 'fish', 'turtle', 'rabbit', 'elephant', 'heart'
     ];
 
     // create cards for every image. A card is defined as a <div> with an <img> element and <div> element
-    for (let i = 1; i <= N; i++) {
+    for (let i = 0; i < NUMBER_OF_IMAGES; i++) {
         const card = document.createElement('div');
         card.className = CARD_CN;
 
         const img = document.createElement('img');
-        const src = i < 10
-            ? `${PATH_PREFIX}00${i}-${origami[i]}.svg`
-            : `${PATH_PREFIX}0${i}-${origami[i]}.svg`;
+        const src = i < 9

```

Comment:
Nit: avoid magic number, and consider declare it at the top of file as a constant as well.

Reply:
good idea, will do that!